internal/server/server.go

```diff
@@ -102,6 +161,12 @@ func (s *Server) StoreHandler(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
 }
 
+// @Summary Download
+// @Description download an attestation
+// @Produce  json
+// @Success 200 {object} dsse.Envelope
+// @Tags attestation
+// @Router /v1/donwload/{gitoid} [post]
```

Comment:
Are you imagining addition operations for `/v1/policies` besides upload and download? Attestations and policies will both be DSSE envelopes stored the same way inside of Archivista.

Reply:
> Are you imagining addition operations for `/v1/policies` besides upload and download? Attestations and policies will both be DSSE envelopes stored the same way inside of Archivista.

Ok, that's true, and I missed this design detail.
We might need Archivista to have some endpoints for TUF "operations." Signing TUF root metadata, rotate/revoke keys, etc.
I think we could keep it as proposed here and add TUF  in a namespace (i.e.,`/v1/tuf/<endpoints>`)
WDYT?

Reply:
I think that's reasonable. Right now my thought process was in-line with John's, that uploading policies would happen through the same mechanisms as attestations since they're both wrapped in DSSE envelopes... and we could possibly store other data signed with DSSE envelopes.

I also agree that adding a tuf namespace makes a ton of sense.